### ISSUE TITLE
Bug: Cannot install maturin from PyPI

### ISSUE THREAD

**Issue by clbarnes** [PyO3/maturin#196]
`pip install maturin` gets

```
ERROR: Could not find a version that satisfies the requirement maturin (from versions: )
No matching distribution found for maturin
```

using pip 19.2.3, python 3.7.3. Same result if I look specifically for version `0.7.0`, but with `from versions: none`. If I try to `pip install .` my package, pip tries to install maturin based on `build-system.requires`, and hits the same error, but with `from versions: 0.7.1b1`.

I can `cargo install` it fine, but then pip can't pick it up from the `pyproject.toml`.

**Comment by clbarnes** [comment]
Now working, oddly. But it automatically installs maturin 0.7.1b1, and cannot find 0.7.0.

**Comment by konstin** [comment]
Deploying the binary wheels for linux and mac from ci failed because I forgot to change the travis secret. I've published 0.7.1-beta.1 to test the fixed token, but there's still some weird build  failure for mac. I'll publish a 0.7.1 once I've figured everything out.

**Comment by clbarnes** [comment]
Great, thanks very much!
